tests/unit_tests/test_wave_06.py

```diff
@@ -33,12 +33,9 @@ def test_get_no_matching_items_by_category():
 
     items = vendor.get_by_category("Electronics")
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+    assert items == []
```

Comment:
👍🏿 To make it more robust you could check to see if the items that were originally in the inventory are still there.